include/email.js

```diff
@@ -28,7 +28,11 @@ module.exports = function EmailServiceModule(pb) {
      * @class EmailService
      * @constructor
      */
-    function EmailService(){}
+    function EmailService(siteUid, onlyThisSite) {
+
+        this.site = pb.SiteService.getCurrentSite(siteUid);
+        this.onlyThisSite  = onlyThisSite;
+    }
```

Comment:
Instead of parameterizing the constructor please use an object with properties.  Then error check and throw when required properties are not provided.  As irritating as it is to have things blow up I'd rather have the error message that tells me why I botched it.